src/metadata.rs

```diff
@@ -104,9 +88,7 @@ fn extract_date_time_exif_field(
                 Ok(date_time) => Ok(Some(date_time)),
                 Err(err) => Err(err),
             },
-            _ => Err(MetadataError {
-                message: "Exif date field is not ascii".to_string(),
-            }),
+            _ => Err(MetadataError::ExifDateNotAscii),
```

Comment:
Same here but maybe not as important, if it's not ascii then probably it won't be really readable :p

Reply:
I think non ascii characters can mess up terminal view, I would prefer to not print non ascii

Reply:
Good point, maybe just the file like you did in the other error.